metron-stellar/stellar-common/README.md

```diff
@@ -1352,6 +1352,61 @@ IS_EMAIL
 [Stellar]>>> 
 ```
 
+#### %timing
+
+Prints out timing information from the last stellar statement executed. The information has the total
+execution time, and also a break down of execution time by stellar function.
+
+%timing supports filter on the timing output.  If you pass one or more strings to %timing, only
+execution times those each of those tags will show.  Any functions that do not have ALL the tags passed will not display
```

Comment:
Should this say 'times with each of those'?

Reply:
right you are, change it i will, yes yes